guidelines/program-guidelines.md

```diff
@@ -0,0 +1,81 @@
+
+## Introductions
+
+If you have an emcee, be sure to communicate with speakers about this well in advance. Ask how they’d like to be introduced — some may provide you with text, some may care less (in which case, prepare the text yourself and make sure the emcee has it well in advance). Make sure you know their pronouns, and have your emcee work with them to make sure their name will be pronounced correctly.
+
+Some speakers feel awkward about self-introduction, so having an emcee introduce them can be quite helpful. Others may have a talk that starts with an introduction that segues into the material, so they may wish for the emcee introduction to be shorter.
+
+Overall, be consistent with how folks are introduced.
+
+## Logisitics and Announcements
+
+A website or central document that describes the general practices and layout of the event is very valuable and can eliminate a lot of redundent questions.
+
+Try to keep all the information for the event in a central and logically organized manner. Keep in mind that weird things *will* happen so have a plan for emergency/urgent announcements (anyone have an HDMI cable?) and make sure people know where to look for these. Use the same medium (twitter/Slack/IRC/etc) as inconsistancy will lead to confusion or missed announcements. 
+
+Visit the venue at least once before the event, and prep the volunteers on the basics (especially if they are also coming in from out of town).
+
+## Things take time
+
+Although we “know” that things take time, we rarely schedule time for all the things, *beyond talks*,  that need to happen during an event.  Some of these things include:
+
+  - Applause (seriously)
+  - Plugging/Unplugging Laptops
+  - Speaker Introductions
+  - Announcements
+
+In order to realistically design your program, you should specifically set aside time to account for these things. We recommend:
+
+  - 10min at the beginning and end of the program day
+  - 5-10min between talks
+
+These times serve 2 purposes:
+
+  - realistically set time aside for things that have to happen and things **take time** 
+  - You can use this time to recover speakers going over and under!
+
+You should not only set this time aside but you should clearly show it on the program you display and distribute to attendees. They appreciate a realistic schedule and will be more likely to comply with it if they know when it is and why it’s there in advance which always makes things run more smoothly.
+
+## Tech Check
+It is extremely useful to have your A/V or organizers available to help speakers set up their machines on stage to test their display *before* they have to speak. Inevitably, not every speaker will be available (and some will flake out and forget) but every person you are able to tech check saves you time in your program and also allows the event to appear more professional. 
```

Comment:
Maybe make it clear to use the breaks (morning break, lunch break, etc) for tech checks.
Also someone should be responsible in the minutes before the talk to setup speaker (mic, laptop)